openprescribing/matrixstore/management/commands/matrixstore_download_practice_stats.py

```diff
@@ -0,0 +1,62 @@
+"""
+Downloads practice statistics data from BigQuery
```

Comment:
Would be good to document exactly where this data is downloaded to.

Reply:
Good point.